hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -88,7 +91,8 @@
 public class AbfsClient implements Closeable {
   public static final Logger LOG = LoggerFactory.getLogger(AbfsClient.class);
   public static final String HUNDRED_CONTINUE_USER_AGENT = SINGLE_WHITE_SPACE + HUNDRED_CONTINUE + SEMICOLON;
-
+  public static final String MD5_ERROR = "The MD5 value specified in the request "
```

Comment:
should be placed in AbfsConstants, so it's more visible, javadocs to explain purpose. and that it comes with a 400

Reply:
Taken